ts/Series/Scatter/ScatterSeries.ts

```diff
@@ -19,6 +19,10 @@
 import type ScatterPoint from './ScatterPoint';
 import type ScatterSeriesOptions from './ScatterSeriesOptions';
 
+// ScatterSeries depends on ColumnSeries and LineSeries, so we must ensure they are loaded
```

Comment:
Should it import actual references that are used?
It is strange that this builds in the first place if some of the references used in code are missing.

Reply:
I agree, we should import specific symbols if possible. However I am not familiar enough with the internal architecture, nor the build system, to make a better suggestion than this. 

I'll gladly review any suggestions though...